### PR TITLE
No Windows support for CLI client

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/QcF8KQZf/63-update-omero-downloads-page-cli-docs-re-no-windows-server-support
Staged at https://www.openmicroscopy.org/site/support/omero5.3-staging/users/cli/index.html and https://www.openmicroscopy.org/site/support/omero5.3-staging/users/cli/installation.html

### CODE REVIEW

File: omero/users/cli/index.txt [hunk 1]
```diff
@@ -5,6 +5,11 @@ The |CLI| is a set of Python_ based system-administration, deployment and
 advanced user tools. Most of commands work remotely so that the |CLI| can be
```
Comment (pwalczysko) [inline omero/users/cli/index.txt:5]: This was not changed in this PR, nevertheless, the brackets around the ``(CLI)`` seem out of place here.

**Comment by pwalczysko** [comment]
One minor comment. Builds and reads fine otherwise. As the comment concerns a line which was not  changed in this PR, possibly merge and open a follow-up PR ? Up to @hflynn

File: omero/users/cli/index.txt (hunk 1 above)
Comment (hflynn) [reply omero/users/cli/index.txt:5]: They aren't brackets, it's mark up which gives mouseover text to define CLI as 'Command line interface' in the webpages

**Comment by pwalczysko** [comment]
Right, sorry, thanks @hflynn  https://github.com/openmicroscopy/ome-documentation/pull/1645#discussion_r104717651, my bad. 

Ready to merge then.

**Pull request merged**